paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -1030,9 +1030,9 @@ internal class PaymentSheetActivityTest {
 
         scenario.launch(intent).onActivity { activity ->
             testDispatcher.scheduler.advanceTimeBy(50)
-            assertThat(activity.buyButton.externalLabel).isEqualTo("Pay")
+            assertThat(activity.buyButton.externalLabel?.resolve(context)).isEqualTo("Pay")
             testDispatcher.scheduler.advanceTimeBy(250)
-            assertThat(activity.buyButton.externalLabel).isEqualTo("Pay CA\$99.99")
+            assertThat(activity.buyButton.externalLabel?.resolve(context)).isEqualTo("Pay CA\$99.99")
```

Comment:
Non-blocking: I'm wondering if we should test these based on resolution or actual `ResolvableString` attributes. So this or:
```kotlin
assertThat(activity.buyButton.externalLabel).isEqualTo(resolvableString(R.string.buy_button_label, "CA\$99.99"))
```